chartfx-math/src/main/codegen/de/gsi/math/MathGen.java

```diff
@@ -736,54 +734,56 @@ public double Voigt(double xx, double sigma, double lg, int r) {
                     k = k + (c[j] * (mq[j] * mf[j] - y0 * ym[j]) + s[j] * yf * xm[j]) / (mq[j] + y0q)
                         + (c[j] * (pq[j] * pf[j] - y0 * yp[j]) - s[j] * yf * xp[j]) / (pq[j] + y0q);
                 }
-                k = y * k + Exp(-xq);
+                k = y * k + exp(-xq);
             }
         }
 
         return k / 2.506628 / sigma; // Normalize by dividing by sqrt(2*pi)*sigma.
     }
 
-    public static double BesselI(int n, double x) {
-        // Compute the Integer Order Modified Bessel function I_n(x)
-        // for n=0,1,2,... and any real x.
-        //
-        // --- NvE 12-mar-2000 UU-SAP Utrecht
+    /**
+    *  Compute the Integer Order Modified Bessel function I_n(x)
+    *  for n=0,1,2,... and any real x.
+    *
+    *  --- NvE 12-mar-2000 UU-SAP Utrecht
+     */
+    public static double besselI(int n, double x) {
```

Comment:
Codacy found an issue: [The method 'besselI(int, double)' has an NPath complexity of 480, current threshold is 200](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)